package.json

```diff
@@ -1,29 +1,32 @@
 {
   "name": "ng-promise-dsl",
   "fileName": "ng-promise-dsl",
-  "version": "0.0.0",
+  "version": "1.0.0",
   "dependencies": {
     "angular": "1.2.21"
   },
   "devDependencies": {
     "angular-mocks": "1.2.21",
-    "grunt": "0.4.5",
-    "grunt-contrib-clean": "0.4.1",
-    "grunt-contrib-concat": "0.3.0",
-    "grunt-contrib-copy": "0.4.1",
-    "grunt-contrib-jshint": "0.8.0",
-    "grunt-contrib-uglify": "0.2.0",
-    "grunt-contrib-watch": "0.4.0",
+    "grunt": "1.0.1",
+    "grunt-contrib-clean": "1.0.0",
+    "grunt-contrib-concat": "1.0.1",
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-contrib-jshint": "1.1.0",
+    "grunt-contrib-uglify": "2.0.0",
+    "grunt-contrib-watch": "1.0.0",
     "grunt-karma": "2.0.0",
-    "grunt-ngmin": "0.0.2",
+    "grunt-ngmin": "0.0.3",
     "grunt-rev": "0.1.0",
     "jasmine-core": "2.5.2",
     "karma": "1.3.0",
     "karma-jasmine": "1.0.2",
-    "karma-phantomjs-launcher": "1.0.2"
+    "karma-phantomjs-launcher": "1.0.2",
+    "phantomjs": "^2.1.7",
+    "underscore": "^1.8.3"
```

Comment:
I should get rid of Grunt altogether, I don't think it's the right approach to have a library supply a minified version of itself anymore in this day and age; modern front-end apps will throw all of their shit together and optimize on the whole package.

As for lodash / underscore, I'd rather get rid of that too; it makes no sense for a 10 kb file (probably like 2 if you disregard the comments) to have that big a dependency.